code/__HELPERS/global_lists.dm

```diff
@@ -172,6 +172,12 @@
 		if(initial(D.name))
 			GLOB.keybindings += new path()
 
+	for(var/path as anything in subtypesof(/datum/objective))
```

Comment:
That would filter for instances of `/datum/objective` and since `subtypesof`/`typesof` returns paths, nothing would be returned. I will however remove the `as anything`, no sure why I added that.